tests/integration/test_orders_api.py

```diff
@@ -337,7 +337,7 @@ async def test_get_order(oid, order_description, session):
 
 
 @respx.mock
-def test_get_order_sync(oid, order_description, session):
+def test_get_order_sync(oid, order_description, ):
```

Comment:
```suggestion
def test_get_order_sync(oid, order_description):
```

nit: remove trailing comma and whitespace character

Reply:
oops, didn't mean to add this - reverted

fwiw, was trying to figure out the cause of "leaks" and while session isn't used here (or in other tests), didn't want to make too much noise in this commit